msteams-platform/tabs/how-to/using-teams-client-sdk.md

```diff
@@ -1,72 +1,469 @@
+| - | - |
+| `initialize` | `app.initialize` |
+| `getContext` | `app.getContext` |
+| `registerOnThemeChangeHandler` | `app.registerOnThemeChangeHandler` |
+
+| Original namespace `appInitialization` | New namespace `app` |
+| - | - |
+| `appInitialization.notifyAppLoaded` | `app.notifyAppLoaded` |
+| `appInitialization.notifySuccess` | `app.notifySuccess` |
+| `appInitialization.notifyFailure` | `app.notifyFailure` |
+| `appInitialization.notifyExpectedFailure` | `app.notifyExpectedFailure` |
+| `appInitialization.FailedReason` enum | `app.FailedReason` |
+| `appInitialization.ExpectedFailureReason` enum | `app.ExpectedFailureReason` |
+| `appInitialization.IFailedRequest` enum | `app.IFailedRequest` |
+| `appInitialization.IExpectedFailureRequest` enum | `app.IExpectedFailureRequest` |
+
+##### *pages* namespace
+
+The `pages` namespace includes functionality for running and navigating webpages within various Microsoft 365 hosts, including Teams, Office, and Outlook. It also includes several subcapabilities, implemented as subnamespaces.
+
+| Original namespace `global (window)` | New namespace `pages` |
+| - | - |
+| `setFrameContext` | `pages.setCurrentFrame` (renamed) |
+| `initializeWithFrameContext` | `pages.initializeWithFrameContext` |
+| `registerFocusEnterHandler` | `pages.registerFocusEnterHandler`
+| `registerFullScreenHandler` | `pages.registerFullScreenHandler` |
+| `navigateCrossDomain` | `pages.navigateCrossDomain` |
+| `returnFocus` | `pages.returnFocus` |
+
+###### *pages.tabs*
+
+| Original namespace `global (window)` | New namespace `pages.tabs` |
+| - | - |
+| `getTabInstances` |  `pages.tabs.getTabInstances` |
+| `getMruTabInstances` | `pages.tabs.getMruTabInstances` |
+| `navigateToTab` | `pages.tabs.navigateToTab` |
+
+| Original namespace `navigation` | New namespace `pages.tabs` |
+| - | - |
+| `navigation.navigateToTab` | `pages.tabs.navigateToTab` |
+
+###### *pages.config*
+
+| Original namespace `settings` | New namespace `pages.config`  |
+| - | - |
+| `settings.setSettings` | `pages.config.setConfig` (renamed)
+| `settings.getSettings` | `pages.config.getConfig` (renamed)
+| `settings.setValidityState`| `pages.config.setValidityState`
+| `settings.initialize` | `pages.config.initialize`
+| `settings.registerOnSaveHandler`| `pages.config.registerOnSaveHandler`
+| `settings.registerOnRemoveHandler` | `pages.config.registerOnRemoveHandler`
+| `settings.Settings` interface | `pages.config.Config` (renamed)
+| `settings.SaveEvent` interface | `pages.config.SaveEvent` (renamed)
+| `settings.RemoveEvent` interface | `pages.config.RemoveEvent` (renamed)
+| `settings.SaveParameters` interface | `pages.config.SaveParameters` (renamed)
+| `settings.SaveEventImpl` interface | `pages.config.SaveEventImpl` (renamed)
+
+| Original namespace `global (window)` | New namespace `pages.config` |
+| - | - |
+| `registerEnterSettingsHandler` | `pages.config.registerChangeConfigHandler` (renamed)
```

Comment:
In v1.12.1, this was renamed back to its original name: `registerChangeSettingsHandler`. Should we reference the most recent name in v1 in this table as the Original API?

Reply:
Yup! Fixed now